includes/class-placements.php

```diff
@@ -618,27 +664,29 @@ public static function render_ad_unit_mock( $provider_id, $ad_unit, $classes = [
 		$width  = count( $sizes ) ? max( array_column( $sizes, 0 ) ) : 300;
```

Comment:
Non-blocking: I totally understand why we show the largest size as the preview, but after #375 the actual front-end behavior is more intelligent than the preview, which can show some unexpected behavior when inserting a multisize unit in a placement inside a bounds container:

<img width="994" alt="Screen Shot 2022-04-14 at 3 07 37 PM" src="https://user-images.githubusercontent.com/2230142/163475400-5b28a9fa-7474-413f-8400-9751759a055b.png">

I'm not sure what the best way to solve for this is, other than recreating some of the bounds container logic to figure out the most appropriate size to show.

Reply:
I don't think we should apply the size mapping rules we build for GAM to the overall provider-agnostic placement preview. It is possible to use the placement data to create a provider-specific preview experience but I think it's an extra feature that we can focus on another time.

Reply:
That makes sense—I'm okay with leaving this for now.